scripts/change_url

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
I'm unsure this is a feature that exists upstream, If this isn't the case, please delete this file

Reply:
Well, even if the upstream app hasn't a change_url functionnality, website can be moved from one domain to another one

Reply:
in theory yes, but it's far from being always true, sometimes it's just not possible.

Here for example, with federation using the domain name, how do you expect to change the domain name? I'm pretty sure it won't work :p

Reply:
I agree with you, the federation part will break if you change the domain name. But even if the federation breaks someone may want to change the name of the domain ^^